### PR TITLE
Feature/3 issue testing

### PR DESCRIPTION
**Pull request by theblockstalk**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+  "include": [
+    "src",
+    "types"
+  ],
   "compilerOptions": {
+    "target": "es5",
     "module": "esnext",
-    "lib": ["dom", "esnext"],
+    "lib": [
+      "dom",
```
Comment (Rebal67) [inline tsconfig.json:11]: why dom ?

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                "--inspect-brk",
+                "${workspaceRoot}/node_modules/tsdx/dist/index.js",
+                "test",
+                "--runInBand"
+            ],
+            "console": "integratedTerminal",
+            "internalConsoleOptions": "neverOpen",
+        }
+    ]
+}
```
Comment (Rebal67) [inline .vscode/launch.json:22]: shouldn't this be ignored ?

Comment (theblockstalk) [reply .vscode/launch.json:22]: we all decided to use VS code which means to me we can share VS code IDE settings and so, so IMO its ok to be here

File: tsconfig.json (hunk 1 above)
Comment (theblockstalk) [reply tsconfig.json:11]: this may be executed in a browser setting

**Pull request merged**